symfony/gatewayapi-notifier/5.3/manifest.json

```
@@ -0,0 +1,5 @@
+{
+    "env": {
+        "#1": "GATEWAYAPI_DSN=gatewayapi://TOKEN@default?from=SENDER"
```

```suggestion
        "#1": "GATEWAYAPI_DSN=gatewayapi://TOKEN@default?from=FROM"
```

To stay consistent with the README